config.go

```diff
@@ -97,6 +98,13 @@ func newConfig(v *viper.Viper, args []string) (*config, error) {
 	var conf config
 
 	intervalStr, _ := flagSet.GetString("interval")
+	isIntervalFlagSet := isFlagSet("interval", flagSet)
```

Comment:
Thank you.

It might be easy to use the environment variable as default value of the flag.

https://github.com/sachaos/viddy/pull/50/files#diff-0e426a43248661127a0c0ee115aef7a1093b635f8993b3f7ebb1dd9f05b8f249R76

Reply:
Is there anything you want to change (you put up a link of diff) or any problem in merging the pull request ?